corehq/apps/cloudcare/utils.py

```diff
@@ -38,6 +38,18 @@ def webapps_module(domain, app_id, module_id):
     return _webapps_url(domain, app_id, selections=[module_id])
 
 
+def filter_available_web_apps(apps, domain, user, is_preview):
```

Comment:
nice, I was thinking it'd be helpful to pull this out for easier testing

Reply:
yeah I'm attempting to write tests for this now